### ISSUE TITLE
CMake warning: policy CMP0042

### ISSUE THREAD

**Issue by stuarteberg** [ukoethe/vigra#245]
On Mac, CMake 3.0 issues the following warning:

```
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   vigraimpex

This warning is for project developers.  Use -Wno-dev to suppress it.
```

The output of `cmake --help-policy CMP0042` is shown below.

CMP0042
-------

``MACOSX_RPATH`` is enabled by default.

CMake 2.8.12 and newer has support for using ``@rpath`` in a target's install
name.  This was enabled by setting the target property
``MACOSX_RPATH``.  The ``@rpath`` in an install name is a more
flexible and powerful mechanism than ``@executable_path`` or ``@loader_path``
for locating shared libraries.

CMake 3.0 and later prefer this property to be ON by default.  Projects
wanting ``@rpath`` in a target's install name may remove any setting of
the ``INSTALL_NAME_DIR`` and ``CMAKE_INSTALL_NAME_DIR``
variables.

This policy was introduced in CMake version 3.0.  CMake version
3.2.2 warns when the policy is not set and uses OLD behavior.  Use
the cmake_policy command to set it to OLD or NEW explicitly.

------

**Comment by ukoethe** [comment]
@stuarteberg: Has this been fixed by 78ed77635754837410c4743c534b2cf4d0f2d3a6?

**Comment by stuarteberg** [comment]
This seems to be fixed now.

For future reference, here's the output I get from cmake 3.5, when configuring with [`ilastik-build-conda/vigra/build.sh`](https://github.com/ilastik/ilastik-build-conda/blob/master/vigra/build.sh):

```
-- The C compiler identification is GNU 4.8.5
-- The CXX compiler identification is GNU 4.8.5
-- Checking whether C compiler has -isysroot
-- Checking whether C compiler has -isysroot - yes
-- Checking whether C compiler supports OSX deployment target flag
-- Checking whether C compiler supports OSX deployment target flag - yes
-- Check for working C compiler: /miniconda/envs/test-relabel/bin/gcc
-- Check for working C compiler: /miniconda/envs/test-relabel/bin/gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Checking whether CXX compiler has -isysroot
-- Checking whether CXX compiler has -isysroot - yes
-- Checking whether CXX compiler supports OSX deployment target flag
-- Checking whether CXX compiler supports OSX deployment target flag - yes
-- Check for working CXX compiler: /miniconda/envs/test-relabel/bin/g++
-- Check for working CXX compiler: /miniconda/envs/test-relabel/bin/g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring VIGRA version 1.11.0
-- Checking template recursion depth: ok
-- Searching for ZLIB
--    in prefix /miniconda/envs/test-relabel
-- Found ZLIB: /miniconda/envs/test-relabel/lib/libz.dylib (found version "1.2.8") 
-- Searching for TIFF
--    in prefix /miniconda/envs/test-relabel
-- Found TIFF: /miniconda/envs/test-relabel/lib/libtiff.dylib (found version "4.0.6") 
-- Searching for JPEG
--    in prefix /miniconda/envs/test-relabel
-- Found JPEG: /miniconda/envs/test-relabel/lib/libjpeg.dylib  
-- Searching for PNG
--    in prefix /miniconda/envs/test-relabel
-- Found PNG: /miniconda/envs/test-relabel/lib/libpng.dylib (found version "1.6.17") 
-- Searching for FFTW3
--    in prefix /miniconda/envs/test-relabel
-- Found FFTW3: /miniconda/envs/test-relabel/lib/libfftw3.dylib  
-- Searching for FFTW3F
--    in prefix /miniconda/envs/test-relabel
-- Found FFTW3F: /miniconda/envs/test-relabel/lib/libfftw3f.dylib  
-- Searching for HDF5
--    in prefix /miniconda/envs/test-relabel
--    Checking HDF5 version (at least 1.8): ok
-- Found HDF5: /miniconda/envs/test-relabel/lib/libhdf5.dylib  
-- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE) 
-- Found PythonInterp: /miniconda/envs/test-relabel/bin/python (found version "2.7.11") 
-- Searching for Boost (at least version 1.55.0)
--    in prefix /miniconda/envs/test-relabel
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - found
-- Found Threads: TRUE  
-- Boost version: 1.55.0
-- Found the following Boost libraries:
--   thread
--   system
--   date_time
--   chrono
--   atomic
-- Checking for threading support:   boost::thread
-- Detected C++ version: 201103
-- Checking VIGRANUMPY_DEPENDENCIES
-- Using Python 2.7.11 at /miniconda/envs/test-relabel/bin/python
-- Found Python includes:  /miniconda/envs/test-relabel/include/python2.7
-- Found Python library: /miniconda/envs/test-relabel/lib/libpython2.7.dylib
-- Found boost_python library: /miniconda/envs/test-relabel/lib/libboost_python-mt.dylib
-- Searching for Python numpy: ok
-- Searching for Python nosetests: ok
-- Searching for sphinx documentation generator: ok
-- Found VIGRANUMPY_DEPENDENCIES: TRUE  
-- ---------------------------------------------------------
-- VIGRA configuration information:
-- ---------------------------------------------------------
--   Using ZLIB  libraries: /miniconda/envs/test-relabel/lib/libz.dylib
--   Using PNG  libraries: /miniconda/envs/test-relabel/lib/libpng.dylib;/miniconda/envs/test-relabel/lib/libz.dylib
--   Using TIFF libraries: /miniconda/envs/test-relabel/lib/libtiff.dylib
--   Using JPEG libraries: /miniconda/envs/test-relabel/lib/libjpeg.dylib
--   OpenEXR disabled by user (WITH_OPENEXR=0)
--   Using FFTW libraries: /miniconda/envs/test-relabel/lib/libfftw3.dylib
--   Using HDF5 libraries: /miniconda/envs/test-relabel/lib/libhdf5.dylib;/miniconda/envs/test-relabel/lib/libhdf5_hl.dylib;/miniconda/envs/test-relabel/lib/libz.dylib
--   Using Boost Graph Library: /miniconda/envs/test-relabel/include/boost/graph
--   LEMON graph library disabled by user (WITH_LEMON=0)
--   Using Python libraries: /miniconda/envs/test-relabel/lib/libpython2.7.dylib;/miniconda/envs/test-relabel/lib/libboost_python-mt.dylib
--   Using Numpy includes:   /miniconda/envs/test-relabel/lib/python2.7/site-packages/numpy/core/include
-- ---------------------------------------------------------
--   building shared lib
--   binaries will be generated in: /Users/bergs/workspace/vigra/build
-- ---------------------------------------------------------
--   includes will be installed at: /miniconda/envs/test-relabel/include
--   libraries will be installed at: /miniconda/envs/test-relabel/lib
--   binaries will be installed at: /miniconda/envs/test-relabel/bin
--   vigra manuals cannot be generated (Doxygen or Python not found)
--   vigranumpy will be installed at /miniconda/envs/test-relabel/lib/python2.7/site-packages
--   vigranumpy manuals will be installed at: /miniconda/envs/test-relabel/doc/vigranumpy/html/index.html
-- ---------------------------------------------------------
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    Boost_PYTHON_LIBRARY_DEBUG
    Boost_PYTHON_LIBRARY_RELEASE
    PYTHON_INCLUDE_DIR
    PYTHON_LIBRARY


-- Build files have been written to: /Users/bergs/workspace/vigra/build
```
